3.normalize_data/normalize_data.py

```diff
@@ -37,21 +39,24 @@
 data_path = pathlib.Path(
     "../1.idr_streams/extracted_features/negative_control_data/merged_features/"
 )
+print("Loading negative control data...")
 data = compile_mitocheck_batch_data(data_path)
+print("Normalizing negative control data...")
 normalized_data = get_normalized_mitocheck_data(data, normalization_scaler)
 # save normalized negative control data
 save_path = pathlib.Path(f"{results_dir}/negative_control_data.csv.gz")
+print("Saving normalized negative control data...")
 normalized_data.to_csv(save_path, compression="gzip", index=False)
 
 # normalize positive control data
-print("Loading")
 data_path = pathlib.Path(
```

Comment:
consider declaring paths at beginning and changing path name variable to be more descriptive
```suggestion
positive_control_data_path = pathlib.Path(
```

Reply:
Done in https://github.com/WayScience/mitocheck_data/pull/35/commits/653872344885dcc3b2a0f4ce24bca87a444a65c5